lib/components/joystick/joystick_directional.dart

```diff
@@ -66,77 +66,59 @@ class JoystickDirectional {
     _tileSize = size / 2;
   }
 
-  void initialize(Vector2 _screenSize, JoystickController joystickController) {
-    this._screenSize = _screenSize;
+  void initialize(Vector2 screenSize, JoystickController joystickController) {
+    _screenSize = _screenSize;
     _joystickController = joystickController;
-    final Offset osBackground =
-        Offset(margin.left, _screenSize.y - margin.bottom);
+
+    final osBackground = Offset(margin.left, _screenSize.y - margin.bottom);
     _backgroundRect = Rect.fromCircle(center: osBackground, radius: size / 2);
 
-    final Offset osKnob =
-        Offset(_backgroundRect.center.dx, _backgroundRect.center.dy);
+    final osKnob = _backgroundRect.center;
```

Comment:
What does os mean in `osKnob`?

Reply:
I haven't got a clue haha

Reply:
I think it's short for offset? maybe. I will just inline that var